single_tech_samples/databricks/sample2_enterprise_azure_databricks_environment/deploy.sh

```diff
@@ -172,27 +214,59 @@ if [[ -z $nsg_arm_output ]]; then
     exit 1
 fi
 
+echo "Deploying Route Table..."
+nsg_arm_output=$(az deployment group create \
```

Comment:
This is not `nsg` so better to change the variable name I guess.

Reply:
Fixed